recipes/gobject-introspection/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "gobject-introspection" %}
+{% set major = "1.51" %}
+{% set patch = "1" %}
+{% set version = major ~ "." ~ patch %}
+{% set sha256 = "2c47e7632df936055b4e96f48315e2d392be10a0642ce11610fba5240e19772d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.xz
+  url: http://ftp.gnome.org/pub/GNOME/sources/{{ name }}/{{ major }}/{{ name }}-{{ version }}.tar.xz
+  sha256: {{ sha256 }}
+  patches:
+    - hugepaths.patch
+
+build:
+  number: 0
+  detect_binary_files_with_prefix: true
+  skip: true  # [win or not py27]
+
+requirements:
+  build:
+    - bison
+    - cairo 1.14.*
+    - flex
+    - gettext  # [osx]
+    - glib 2.51.*
+    - libffi 3.2.*
+    - pkg-config  # [not win]
```

Comment:
As per my other response, I figured I'd keep the selector in in case Windows support is added in the future.